gateway/src/api/latest.rs

```diff
@@ -489,10 +490,22 @@ impl ApiBuilder {
 
     pub fn with_auth_service(mut self, auth_uri: Uri) -> Self {
         let auth_public_key = AuthPublicKey::new(auth_uri.clone());
+
+        let cache = Arc::new(RwLock::new(TtlCache::new(1000)));
```

Comment:
Can we move this line to a `CacheManager::new()`?

Reply:
Yup!